cli/plugins/spark.py

```diff
@@ -4,27 +4,29 @@
 import core
 
 
-@task
-def lambda_example_hive(c):
+@task(autoprint=True)
+def lambda_example_hive(c, json_output=False, month="01"):
     """SUM(trip_distance) GROUP_BY payment_type with preliminary CREATE EXTERNAL TABLE"""
     sql = f"""
-CREATE EXTERNAL TABLE nyc (trip_distance FLOAT, payment_type STRING) 
-STORED AS PARQUET LOCATION 's3a://{core.bucket_name(c)}/nyc-taxi/2019/01/';
+CREATE EXTERNAL TABLE nyc{month} (trip_distance FLOAT, payment_type STRING) 
```

Comment:
❓ if we try to make the testbench even shouldn't we try to call the table always the same? Once again would be clearer for the user/reader

Reply:
Made the change to align the table names (only spark and databend, right?). But given that these queries are kind of self contained, I don't think that the temporary table names make a huge difference.

Reply:
I agree is just to avoid raising questions on the user on why we do it on some engines and not on others